apps/demos/Demos/SelectBox/Overview/Angular/app/app.component.css

```diff
@@ -8,17 +8,14 @@
   min-height: 30px;
 }
 
-::ng-deep .dx-dropdowneditor-input-wrapper .custom-item > img {
+::ng-deep .custom-addon {
   padding-left: 8px;
+  display: flex;
+  align-items: center;
 }
 
-::ng-deep .custom-item .product-name {
-  display: inline-block;
-  padding-left: 50px;
-  text-indent: 0;
-  line-height: 30px;
-  font-size: 15px;
-  width: 100%;
```

Comment:
previously these styles were used for templated items as well. Shouldn't we keep it?